recipes/csv-parser/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CsvParserConan(ConanFile):
+    name = "csv-parser"
+    description = "Vince's CSV Parser with simple and intuitive syntax"
+    topics = ("conan", "csv-parser", "csv", "rfc 4180", "parser", "generator")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/vincentlaucsb/csv-parser"
+    license = "MIT"
+    settings = "os", "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        tools.check_min_cppstd(self, 11)
+
+        compiler = self.settings.compiler
+        compiler_version = tools.Version(self.settings.compiler.version)
+        if compiler == "gcc" and compiler_version < "7":
+            raise ConanInvalidConfiguration("gcc version < 7 not supported")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst="include", src=os.path.join(self._source_subfolder, "single_include"))
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "CSVParser"
+        self.cpp_info.names["cmake_find_package_multi"] = "CSVParser"
+
+        if self.settings.os == "Linux" or self.settings.os == "FreeBSD":
```

Comment:
I don't see the cmake build script installing any cmake scripts.
We only model the "official" targets of an upstream project.
```suggestion
    def package_info(self):
        if self.settings.os == "Linux" or self.settings.os == "FreeBSD":
```

Reply:
Done